hicexplorer/hicFindTADs.py

```diff
@@ -764,7 +623,22 @@ def get_domains(boundary_list):
     return domain_list
 
 
-def save_bedgraph_matrix(outfile, chrom, chr_start, chr_end, score_matrix):
+def toBytes(s):
```

Comment:
@fidelram maybe you want to use six here? http://www.programcreek.com/python/example/54051/six.BytesIO